server/i18n/en.json

```diff
@@ -8749,7 +8749,7 @@
   },
   {
     "id": "model.config.is_valid.saml_idp_descriptor_url.app_error",
-    "translation": "Identity Provider Issuer URL must be a valid URL and start with http:// or https://."
+    "translation": "Identity Provider Issuer URL must be a valid URI."
```

Comment:
note: it's weird to have the config key name containing URL but requiring a URI, but we'll address renaming the config option in a second time, renaming the config key means working on the depreciation of the old key etc... which is out of scope for this PR that we might want merged asap

Reply:
Is it worth filing a ticket for that?

Reply:
Yeah it's on my todo list for today!